src/auth/awss3/core/qgsauthawss3method.h

```diff
@@ -0,0 +1,77 @@
+/***************************************************************************
+  qgsauthawss3method.h
+  --------------------------------------
+  Date                 : December 2022
+  Copyright            : (C) 2022 by Jacky Volpes
+  Email                : jacky dot volpes at oslandia dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#ifndef QGSAUTHAWSS3METHOD_H
+#define QGSAUTHAWSS3METHOD_H
+
+#include <QObject>
+#include <QMutex>
+
+#include "qgsauthconfig.h"
+#include "qgsauthmethod.h"
+#include "qgsauthmethodmetadata.h"
+
+
+class QgsAuthAWSS3Method : public QgsAuthMethod
```

Comment:
```suggestion
class QgsAuthAwsS3Method : public QgsAuthMethod
```

QT convention is to camel-case acronyms too.